openedx/adg/lms/webinars/tests/test_helpers.py

```diff
@@ -199,3 +204,71 @@ def test_remove_emails_duplicate_in_other_list(emails, reference_emails, expecte
     Test that only the list of emails that are not present in reference list of emails are returned
     """
     assert sorted(remove_emails_duplicate_in_other_list(emails, reference_emails)) == sorted(expected_emails)
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'template, msg_id_field_name', [
+        (MandrillClient.WEBINAR_TWO_HOURS_REMINDER, STARTING_SOON_REMINDER_ID_FIELD_NAME),
+        (MandrillClient.WEBINAR_ONE_WEEK_REMINDER, ONE_WEEK_REMINDER_ID_FIELD_NAME),
+    ]
+)
+def test_save_scheduled_reminder_ids(template, msg_id_field_name):
+    """
+    Tests save_scheduled_reminder_ids method is saving reminder ids properly.
+    """
+    registration = WebinarRegistrationFactory()
+    mandrill_response = [
+        {
+            '_id': FAKE_MANDRILL_MSG_ID,
+            'email': registration.user.email
+        }
+    ]
+
+    save_scheduled_reminder_ids(mandrill_response, template, registration.webinar.id)
+
+    registration.refresh_from_db()
+    assert getattr(registration, msg_id_field_name) != ''
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'msg_id_field_name, mock_id', [
+        (STARTING_SOON_REMINDER_ID_FIELD_NAME, ''),
+        (ONE_WEEK_REMINDER_ID_FIELD_NAME, FAKE_MANDRILL_MSG_ID),
+    ]
+)
+def test_cancel_webinar_reminders(msg_id_field_name, mock_id, mocker):
```

Comment:
would it be better to rename mock_id to something like msg_id etc. as it is not actually being mocked using patch or mocker